src/PhpEnvironment/Response.php

```diff
@@ -96,7 +96,6 @@ public function sendHeaders()
             header($header->toString());
         }
 
-        $this->headersSent = true;
```

Comment:
Is this on the parent class?

Reply:
No, it's not used anywhere, it's just a dynamic assignment with no uses, there is a method for that